Fix the startx manpage to be more correct for our setup; this patch by Branden
Robinson.

---
 startx.man |   54 ++++++++++++++++++++++--------------------------------
 1 file changed, 22 insertions(+), 32 deletions(-)

--- a/man/startx.man
+++ b/man/startx.man
@@ -76,6 +76,28 @@ startx -- -layout Multihead
 .RE
 #ifdef __SCOMAN__
 .PP
+Note that in the Debian system, what many people traditionally put in the
+.I .xinitrc
+file should go in
+.I .xsession
+instead; this permits the same X environment to be presented whether
+.IR startx ,
+.IR xdm ,
+or
+.I xinit
+is used to start the X session. All discussion of the
+.I .xinitrc
+file in the
+.IR xinit (1)
+manual page applies equally well to
+.IR .xsession.
+Keep in mind that
+.I .xinitrc
+is used only by
+.IR xinit (1)
+and completely ignored by
+.IR xdm (1).
+.PP
 To determine the client to run,
 .B startx
 looks for the following files, in order:
@@ -134,38 +156,6 @@ and
 files are found in the
 .I __xinitdir__
 directory.
-.PP
-The
-.I .xinitrc
-is typically a shell script which starts many clients according to the
-user's preference.  When this shell script exits,
-.B startx
-kills the server and performs any other session shutdown needed.
-Most of the clients started by
-.I .xinitrc
-should be run in the background.  The last client should run in the
-foreground; when it exits, the session will exit.  People often choose
-a session manager, window manager, or \fIxterm\fP as the ''magic'' client.
-.SH EXAMPLE
-.PP
-Below is a sample \fI\.xinitrc\fP that starts several applications and
-leaves the window manager running as the ''last'' application.  Assuming that
-the window manager has been configured properly, the user
-then chooses the ''Exit'' menu item to shut down X.
-.sp
-.in +4
-.nf
-xrdb \-load $HOME/.Xresources
-xsetroot \-solid gray &
-xbiff \-geometry \-430+5 &
-oclock \-geometry 75x75\-0\-0 &
-xload \-geometry \-80\-0 &
-xterm \-geometry +0+60 \-ls &
-xterm \-geometry +0\-100 &
-xconsole \-geometry \-0+0 \-fn 5x7 &
-exec twm
-.fi
-.in -4
 .SH "ENVIRONMENT VARIABLES"
 .TP 25
 DISPLAY
